### ISSUE TITLE
Send POST request to _changes

### ISSUE THREAD

**Issue by andrelramos** [cloudant/python-cloudant#296]
I'm using filters to get some changes on my server, but my team needed create filters that receives arguments, like the below function:

```jsx
function (doc, req) { 
    
     if(doc.doc_type === req.query.doc_type) {
     
     return true;
  }
  
  return false;
}
```

On python code, i'm calling this function using changes method like this:

```python
client[database].changes(filter='filters/getByDocType')
```

I didn't find a parameter on changes method where i can pass my filter data that is a str value or a dict like a json. On couchdb docs, to do it i need use a POST request and send the filter data on request body. So how can i call some thing like ```.addPostParams(**params)``` ? Check out the couchdb docs about it http://docs.couchdb.org/en/2.0.0/api/database/changes.html#post--db-_changes

Informations about my environment

- python-cloudant version: 2.4.0
- Python version: 3.6

**Comment by alfinkel** [comment]
The `changes` database method returns a [Feed](https://github.com/cloudant/python-cloudant/blob/master/src/cloudant/feed.py) object which in turn performs a GET, so unfortunately this library at present doesn't support the use of a filter as part of the query string on a `_changes` feed with a POST request.  I've marked this as an enhancement request so that we can add this functionality at a later date.  In the short term you either would need to handle your request directly via the `r_session` attribute or you could attempt to subclass Feed and override the [_start](https://github.com/cloudant/python-cloudant/blob/master/src/cloudant/feed.py#L82-L89) method to suit your purposes and handle your requests using a POST.

**Comment by ricellis** [comment]
We won't fix this here.

We're trying to add this in our new [cloudant-python-sdk](https://github.com/IBM/cloudant-python-sdk/)(beta) with the [`post_changes` operation](https://cloud.ibm.com/apidocs/cloudant?code=python#postchanges) but we're waiting on a server side fix for https://github.com/apache/couchdb/issues/3086.
